cmd/kubeadm/app/master/manifests.go

```diff
@@ -244,7 +243,7 @@ func getComponentCommand(component string, s *kubeadmapi.KubeadmConfig) (command
 			"--insecure-experimental-approve-all-kubelet-csrs-for-group=system:kubelet-bootstrap",
 		},
 		scheduler: {
-			// TODO(phase1+): consider adding --address=127.0.0.1 in order to not expose the scheduler port to the rest of the world
+			"--address=127.0.0.1",
```

Comment:
I am not sure about adding this, per Mike's comment https://github.com/kubernetes/kubernetes/pull/33262#discussion_r80314282.